src/pinning.js

```diff
@@ -56,8 +56,13 @@ class Pinning {
           })
         const profileFromPubStore = address => {
           const profile = this.openDBs[address].db.all()
-          let parsedProfile = {}
-          Object.keys(profile).map(key => { parsedProfile[key] = profile[key].value })
+          const parsedProfile = {}
+
+          Object.entries(profile)
```

Comment:
Is this really needed? Isn't the format already `{ value, timeStamp }`?

Reply:
There's a unusual capitalization on timeStamp, I return `{ value, timestamp }`

Reply:
Oh, I see you did the same thing in the js lib as well. Is it really worth the extra complexity though?

Reply:
(discussed on discord: same time complexity as before)